benchmark/benchmark/base.py

```diff
@@ -49,13 +51,13 @@ def __init__(self, argv: List[Any]) -> None:
             "--num_gpus",
             type=int,
             default=1,
-            help="number of GPUs to use. If num_gpus > 0, spark-rapids-ml will run with the number of dataset partitions equal to num_gpus.",
+            help="number of GPUs to use. If num_gpus > 0, will run with the number of dataset partitions equal to num_gpus.",
         )
         self._parser.add_argument(
             "--num_cpus",
             type=int,
             default=6,
-            help="number of CPUs to use. If num_cpus > 0, spark will run and with the number of dataset partitions to num_cpus.",
+            help="number of CPUs to use. If num_cpus > 0, will run with the number of dataset partitions equal to num_cpus.",
```

Comment:
I don't think repartitioning is done for cpu.  Let's delete the repartitioning part of the help.